@ndlib/gatsby-theme-marble/src/components/Shared/ActionModal/index.js

```diff
@@ -56,12 +55,7 @@ const ActionModal = ({
         </button>
       </div>
       <BaseStyles>
-        <div sx={sx.container}>
-          <div sx={bodyStyle(!!footer)}>{children}</div>
-          {footer && (
-            <div sx={sx.footer}>{footer}</div>
-          )}
-        </div>
+        <div sx={sx.content}>{children}</div>
```

Comment:
The changes to this file were recently added. Make sure this didn't get squashed from a rebase.